TodoListApplication/TodoListSPA/src/app/app-config.ts

```diff
@@ -0,0 +1,85 @@
+import { Configuration } from 'msal';
+import { MsalAngularConfiguration } from '@azure/msal-angular';
+
+// checks if the app is running on IE
+export const isIE = window.navigator.userAgent.indexOf('MSIE ') > -1 || window.navigator.userAgent.indexOf('Trident/') > -1;
+
+/** =================== REGIONS ====================
+ * 
+ * 1) Web API configuration parameters
+ * 2) Authentication configuration parameters
+ * 3) MSAL-Angular specific configuration parameters
+ * 
+ * ================================================= 
+*/ 
+
+
+// #region 1) Web API Configuration
+/** 
+ * Enter here the coordinates of your Web API and scopes for access token request
+ */
+export const apiConfig: {scopes: string[], webApi: string} = {
+    scopes: ['Enter_API_Scopes_Here'],
+    webApi: 'Enter_API_Base_Address_Here',
```

Comment:
Enter Api scopes as declared in the app registration 'Expose an Api' blade. like 'api://d8a11510-296e-40bd-b149-e7835acb5ecb/access_as_user'
Enter the TodoList Web API's base address, like "[https://localhost:44351/api/todolist](https://localhost:44351/api/todolist)"